src/java.com/homework5/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package homework5;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Calculator {
+    public void calculate(double a, double b, char c) {
+        List<Double> values = new ArrayList<Double>();
```

Comment:
from java 7 you can use diamond operator

Reply:
do you really need use List for two values? using 2 methods for calculating just make your code complicated